app/controllers/rents_controller.rb

```diff
@@ -1,12 +1,16 @@
 class RentsController < ApplicationController
+  include Pundit
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
   before_action :authenticate_user!, only: %i[index create]
 
   def index
-    render_paginated Rent.all
+    rents = policy_scope(Rent)
```

Comment:
What is `policy_scope(Rent)`?

Reply:
Read Pundit docs Lucas: https://github.com/varvet/pundit#scopes

Reply:
jajajjajajajajja :raising_hand_man: my bad, i've learn it